### ISSUE TITLE
some bugs

### ISSUE THREAD

**Issue by tlyau62** [tlyau62/UnityAddon3#10]
1. wrong debug msg
```
NexTore.Core.Cryptography required a bean of type 'NexTore.Core.ICryptoKeyManager' that could not be found.
```

2. circular dep: use reverse arrow

**Comment by tlyau62** [comment]
2 is fixed at 3d8809ea1e6bb4253c0a01a997ca02427624bcd5

**Comment by tlyau62** [comment]
Found a serious bug on no such bean exception msg.
Added a test case on commit 8887aac8180dddbf45eb98c188110f64755bedbe.
Will fix soon.

**Comment by tlyau62** [comment]
2 is fixed on commit a1423e6a9a3937011e064e8b45371c80daf254b4
